packages/astro/src/runtime/server/render/astro/factory.ts

```diff
@@ -20,29 +16,6 @@ export function isAstroComponentFactory(obj: any): obj is AstroComponentFactory
 	return obj == null ? false : obj.isAstroComponentFactory === true;
 }
 
-// Calls a component and renders it into a string of HTML
-export async function renderToString(
```

Comment:
This old `renderToString` API is moved to `render.ts` and along side the `renderToReadableStream` API.